src/clients/MultiCallerClient.ts

```diff
@@ -513,3 +514,138 @@ export class MultiCallerClient {
     }
   }
 }
+
+export class TryMulticallClient extends MultiCallerClient {
+  override _buildMultiCallBundle(transactions: AugmentedTransaction[]): AugmentedTransaction {
+    const txn = super._buildMultiCallBundle(transactions);
+    txn.method = "tryMulticall";
+    return txn;
+  }
+
+  protected override async _executeTxnQueue(
+    chainId: number,
+    txns: AugmentedTransaction[] = [],
+    _valueTxns: AugmentedTransaction[] = [],
+    simulate = false
+  ): Promise<TransactionResponse[]> {
+    // Lint Appeasement
+    _valueTxns;
```

Comment:
Just to be on the safe side, because this would indicate a misuse of the class.

```suggestion
    assert(_valueTxns.length === 0);
```